src/server/seeds/dev/008_00_voyages.js

```diff
@@ -0,0 +1,58 @@
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex("voyages")
+    .del()
+    .then(function() {
+      // Inserts seed entries
+      return knex("voyages").insert([
+        {
+          id: 1,
+          vessel_id: 1,
+          etd: "2019-12-20",
+          eta: "2018-11-23",
```

Comment:
Would be nice with some more voyages along with more vessels. Make sure that they don't all have the same etd and etas like now. And make sure that etd is always before eta (you can't arrive before you depart).